File: TravelBois2/WebApp/Models/Aerodrom.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WebApp.Models
{
    public class Aerodrom
    {
        [Key]
        [Required]
        public string Grad { get; set; }
        [Required]
        public string Drzava { get; set; }
    }
}

File: TravelBois2/WebApp/ClientApp/src/app/entities/objects/aerodrom.ts
export class Aerodrom {
  id: number;
  grad: string;
  drzava: string;
  aviokompanija: string;

  constructor(gr: string, dr: string, a:string) {
    this.grad = gr;
    this.drzava = dr;
    this.aviokompanija = a;
  }
}
